modules/nerd-fonts/nerd-fonts.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+set -oue pipefail
+
+NERD_FONTS_DIR=/usr/share/fonts/nerd-fonts
+COMPRESS_TYPES=("tar.xz" "zip")
+
+mkcd() {
```

Comment:
I don’t quite understand the purpose of this function, since it’s not hard to type mkdir and cd separetely and I don’t get why we should delete the directory we’re mkcd’ing to if it already exists, wiping the content which the user might want to keep,

Reply:
I get that it reduces repetition, but it also increases abstraction. It's fine, I wont make you remove it, I just usually rather repeat myself if the code that I am repeating is small and easily understandable. I get DRY (dont repeat yourself), but sometimes it's ok to repeat yourself.

You didn't answer why an existing directory is deleted, though.